diff --git a/src/render/SDL_render.c b/src/render/SDL_render.c
index 8cd3a7b..eacc7e2 100644
--- a/src/render/SDL_render.c
+++ b/src/render/SDL_render.c
@@ -89,9 +89,7 @@ static const SDL_RenderDriver *render_drivers[] = {
 #if SDL_VIDEO_RENDER_OGL_ES
     &GLES_RenderDriver,
 #endif
-#if SDL_VIDEO_RENDER_DIRECTFB
     &DirectFB_RenderDriver,
-#endif
 #if SDL_VIDEO_RENDER_METAL
     &METAL_RenderDriver,
 #endif
